recipe/meta.yaml

```diff
@@ -21,21 +21,22 @@ requirements:
     - python
     - pip
     - setuptools
+    - setuptools_scm >=3.4.3
     - wheel
   run:
+    - importlib-metadata <4.3  # flake 8 requires an older version of importlib-metadata
     - autopep8 >=1.6.0,<1.7.0
     - flake8 >=4.0.0,<4.1.0
     - jedi >=0.17.2,<0.19.0
-    - mccabe >=0.6.0,<0.7.0
-    - pluggy
+    - mccabe >=0.6.0,<0.7.0  # flake 8 requires a version less than 0.7.0
```

Comment:
This comment and the `importlib-metadata` suggest that our flake8 package is buggy. Any fixes should be made there, not here.

Reply:
Made fixes to `flake8` and commented out the modifications in the package. There might be some value on keeping the comments for future reference.
https://github.com/AnacondaRecipes/python-lsp-server-feedstock/blob/d4289ed4722d5f28454d7529c5cd15128d7c3f70/recipe/meta.yaml#L26&&L32

Rebuild `flake8` version `4.0.1` to build `python-lsp-server`
https://github.com/AnacondaRecipes/flake8-feedstock/pull/7